pkg/agent/route/route_windows.go

```diff
@@ -35,6 +35,11 @@ const (
 	outboundFirewallRuleName = "Antrea: accept packets to local Pods"
 )
 
+var (
+	GlobalVirtualGWIP    = net.ParseIP("169.254.169.253").To4()
```

Comment:
If it used on local Node only, probably remove "global". Should we name it svcVirtualGWIP?

And, please add some comments to explain its usage.

Reply:
Updated.